cedar-policy-validator/src/human_schema/test.rs

```diff
@@ -53,19 +60,16 @@ mod demo_tests {
         let src = r#"
         action "Foo" appliesTo { context: {} };
         "#;
-        let (schema, _) = SchemaFragment::from_str_natural(src).unwrap();
-        let foo = schema.0.get(&None).unwrap().actions.get("Foo").unwrap();
-        assert_matches!(
-            foo,
-            ActionType {
-                applies_to: Some(ApplySpec {
-                    resource_types: None,
-                    principal_types: None,
-                    ..
-                }),
-                ..
-            }
-        );
+        match SchemaFragment::from_str_natural(src) {
+            Ok(_) => panic!("Should have failed to parse!"),
+            Err(e) => expect_err(
```

Comment:
nit: this could be written with `assert_matches`
```suggestion
        assert_matches!(SchemaFragment::from_str_natural(src), Err(e) => {
            expect_err(
```

Reply:
Similarly in other locations in this file

Reply:
Not anymore that it uses `expect_err`

Reply:
does my suggestion not work?